app/modules.go

```diff
@@ -135,6 +138,7 @@ var moduleAccountPermissions = map[string][]string{
 	valsetpreftypes.ModuleName:               {authtypes.Staking},
 	poolmanagertypes.ModuleName:              nil,
 	cosmwasmpooltypes.ModuleName:             nil,
+	auctiontypes.ModuleName:                  nil, // initialize a module account
```

Comment:
just to make sure, do we not need any module perms here?

Reply:
No we don't! Just need to create the module account :smile: